### ISSUE TITLE
[Docs] Relationship with Durable Task Framework?

### ISSUE THREAD

**Issue by fowl2** [Azure/azure-functions-durable-extension#426]
aka https://github.com/Azure/durabletask

Is this project a fork, a wrapper, etc?

I read the whole readme and a quick skim of the code and can't quite figure it out.

Thanks!

**Comment by cgillum** [comment]
Yes, it's a wrapper. The benefits of this over the core DTFx framework is that you get full integration with Azure Functions, including bindings, Serverless hosting, application insights logging, etc.

Any changes that were made to the core DTFx framework for the benefit of this project (including the DurableTask.AzureStorage provider and a variety of core performance improvements) have been contributed back and are available to non-Functions customers.

**Comment by cecilphillip** [comment]
Would it make sense for us to create some additional documentation and samples are during the **raw** durable task framework ?

**Comment by YunchuWang** [comment]
Closing this as completed because the current documentation now directly explains this relationship.

The [Choose your Durable Task hosting model](https://learn.microsoft.com/azure/durable-task/common/choose-orchestration-framework) article describes Durable Functions as the Azure Functions hosting model and the modern Durable Task SDKs as the standalone/self-hosted model. Both provide the same core durable-execution concepts, while Durable Functions adds Azure Functions hosting integration such as triggers and bindings, managed/serverless scaling, built-in management HTTP APIs, Application Insights integration, and additional storage-provider choices.

The same article also explicitly clarifies the original [Azure/durabletask](https://github.com/Azure/durabletask) Durable Task Framework (DTFx): it is the underlying open-source .NET library used as a dependency by Durable Functions. The docs recommend Durable Functions or the modern standalone Durable Task SDKs for new projects. This resolves the documentation gap described here.
